app/models/concerns/export/exporter.rb

```diff
@@ -60,7 +60,8 @@ def zip_archive
       Zip::File.open(zip_path, Zip::File::CREATE) do |zipfile|
         files.each do |filename|
           zipfile.add(
-            File.join(@export.export_id, filename),
+            # File.join(@export.export_id, filename),
+            File.join(filename), # add without path
```

Comment:
Does this do anything? I think it can be just `filename`

Reply:
Why yes, yes it is!